src/components/SearchHistory.jsx

```diff
@@ -9,8 +9,12 @@ export default function SearchHistory(props) {
   };
 
   const handleClose = (event) => {
-    if (event.currentTarget.innerText != '') 
-      props.getFeed({target: {elements: {feed_url: {value: event.currentTarget.innerText}}}});
+    if (event.currentTarget.innerText !== "")
```

Comment:
We can change the if statement to `if (event.currentTarget.innerText)`

Reply:
The same as https://github.com/amcquade/quick-feed/pull/43#discussion_r720011990